apps/frontend/src/app/components/candidatePicker/CandidatePicker.tsx

```diff
@@ -0,0 +1,96 @@
+import { Candidate, Insight, Trait } from '@innbyggerpanelet/api-interfaces';
+import { FC } from 'react';
+import { Label, Detail } from '@navikt/ds-react';
+
+import style from './CandidatePicker.module.scss';
+import { isUndefined } from 'util';
+
+interface IProps {
```

Comment:
This should have a more describing name

Reply:
I assume this is about IProps. I disagree, as the interface is not exported the fact that it is named "props" and in the same file as the component should make the correlation clear. Unless there are some other advantages I'm not aware of? 😛